src/models/document/document-content.ts

```diff
@@ -17,8 +17,7 @@ import { safeJsonParse, uniqueId } from "../../utilities/js-utils";
 import { comma, StringBuilder } from "../../utilities/string-builder";
 import { SharedModel, SharedModelType } from "../shared/shared-model";
 import { SharedModelUnion } from "../shared/shared-model-manager";
-import { DocumentModelType, IDocumentEnvironment } from "./document";
-import { SectionModelType } from "../curriculum/section";
+import { IDocumentEnvironment } from "./document";
```

Comment:
This is also a circular dependency which can easily be dealt with by putting the `IDocumentEnvironment` into its own file. This is another change that I was thinking of making locally, but it fits nicely with this PR as well.

Reply:
@kswenson Did you get it work locally? The `IDocumentEnvironment` depends on `AppConfigModelType` which is in `app-config-models.ts` which imports stuff from `document-content.ts`. I don't see an easy way to remove this circular reference, so I think it is better to punt it.

Reply:
You're right that separating `IDocumentEnvironment` into its own file doesn't eliminate the circular dependency, but it does eliminate the parent dependency on `document.ts`, so I think it's still worth doing, but it needn't be part of this PR.